README.md

```diff
@@ -1 +1,78 @@
-# ocw-course-hugo-starter
\ No newline at end of file
+# ocw-course-hugo-starter
+
+A starter template for a Hugo site that uses [`ocw-course-hugo-theme`](https://github.com/mitodl/ocw-course-hugo-theme)
+
+![image](https://user-images.githubusercontent.com/12089658/102410491-9b7e1680-3fbe-11eb-9746-e5ae996add40.png)
+
+## Prerequisites
+
+ - NodeJS
+ - Hugo
+
+## Building JS & CSS
+
+To build the JS and CSS for the theme, make sure you have the above prerequisites installed and run the following:
+
+```sh
+./build.sh
+```
+
+## Running the site
+
+To preview your site, run the following command:
+
+```sh
+hugo server
+```
+
```

Comment:
How does one connect to the hugo server?

Reply:
it serves the site at `localhost:1313` (or you can set an argument on the command line to use a different port if you want). the address is printed in the console after you run `hugo server`

Reply:
I added some details about this, let me know if it makes sense.